lib/evaluate_board.rb

```diff
@@ -1,26 +1,27 @@
 # frozen_string_literal: true
 
 class EvaluateBoard
+  WINNING_COMBINATIONS = [
+    [0, 1, 2],
+    [3, 4, 5],
+    [6, 7, 8],
+    [0, 3, 6],
+    [1, 4, 7],
+    [2, 5, 8],
+    [0, 4, 8],
+    [2, 4, 6]
+  ].freeze
+
   def initialize(board_gateway)
     @board_gateway = board_gateway
-    @winning_combinations = [
-      [0, 1, 2],
-      [3, 4, 5],
-      [6, 7, 8],
-      [0, 3, 6],
-      [1, 4, 7],
-      [2, 5, 8],
-      [0, 4, 8],
-      [2, 4, 6]
-    ]
   end
 
   def execute
-    return 'Player one wins' if win_for_player?('X')
-    return 'Player two wins' if win_for_player?('O')
-    return 'Game over' if game_over?
+    return :"Player one wins" if win_for_player?('X')
+    return :"Player two wins" if win_for_player?('O')
+    return :"Game over" if game_over?
 
-    ''
+    :Continue
```

Comment:
Ruby style - `:Continue` should be `:continue` so all lowercase.

Reply:
We are using this format because we are printing these on console and this way the easiest(lazy) way to convert them to human readable string, we will find a better way to do it :)